manifest.yaml

```
@@ -26,6 +26,11 @@ add-commit-metadata:
 packages:
  - microdnf
 
+# Scripts for opting into staying on iptables-legacy after migration. Lower into
+# `fedora-coreos-base.yaml` once we're ready to migrate `testing`.
+ostree-layers:
+  - overlay/35coreos-iptables
```

```suggestion
  - overlay/35next-coreos-iptables
```

Maybe we try to make it clear in our naming scheme when manifests are going to not apply broadly. We can move it once they do?

---

Mind if we don't do this rename? I don't want to invalidate my testing.

---

Yeah, that's fine.